src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -171,3 +173,16 @@ Napi::Value ModelWrap::get_output_shape(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+Napi::Value ModelWrap::get_ops(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    std::vector<std::shared_ptr<ov::Node>> ops = _model->get_ops();
+
+    Napi::Array result = Napi::Array::New(env, ops.size());
+    for (size_t i = 0; i < ops.size(); ++i) {
+        Napi::Object nodeWrap = NodeWrap::get_class(env).New({Napi::External<std::shared_ptr<ov::Node>>::New(env, &ops[i])});
```

Comment:
If you create js object like this, you will kind of create a new identical class and you won't be able to compare them using `instanceof` operator. 
To avoid that you can use previously created `Napi::Function` from `get_class`. It may look similar to this:
https://github.com/openvinotoolkit/openvino/blob/dc3eaf0a2b816fc32a59e79455bce33ec54f535c/src/bindings/js/node/src/helper.cpp#L255-L264